src/wp-includes/update.php

```diff
@@ -29,7 +29,7 @@
  *                           Defaults to false, true if $extra_stats is set.
  */
 function wp_version_check( $extra_stats = array(), $force_check = false ) {
-	global $wpdb, $wp_local_package;
+	global $wpdb, $wp_local_package, $table_prefix;
```

Comment:
$table_prefix isn't used in the query :)

Reply:
+1